settings.gradle.kts

```diff
@@ -42,10 +42,10 @@ dependencyResolutionManagement {
             library("kotest-assertions-json", "io.kotest", "kotest-assertions-json").versionRef("kotest")
             library("kotest-property", "io.kotest", "kotest-property").versionRef("kotest")
             library("kotest-framework-datatest", "io.kotest", "kotest-framework-datatest").versionRef("kotest")
-            library("kotest-wiremock", "io.kotest.extensions:kotest-extensions-wiremock:3.0.1")
 
             library("mockito", "org.mockito.kotlin:mockito-kotlin:5.2.1")
             library("archunit", "com.tngtech.archunit:archunit-junit5:1.2.1")
+            library("wiremock", "com.github.tomakehurst:wiremock:3.0.1")
```

Comment:
Why we removing autoconfigured wiremock for kotest and placed regular version?

Reply:
i just thought that if i we use only one class from kotest-wiremock and it is originaly from com.github.tomakehurst:wiremock:3.0.1 it would be cleaner to use the original dependency.

Reply:
but i can restore kotest-wiremock if u want

Reply:
IMHO i will leave as was before